### PR TITLE
Add more development flavor texts

### PR DESCRIPTION
**Pull request by polypixeldev**
## Summary of the problem

This change is being made to a couple more flavor texts to the HCB homepage, specifically for the default set of flavor texts and the development mode set of flavor texts. It helps to solve the problem of there not being enough flavor texts on the HCB homepage, although this problem will never be fully solved.

## Describe your changes

I added a couple lines to `flavor_text_service.rb` which include text that will be displayed for users when they navigate to the HCB homepage. One line is for the set of development flavor texts, and the other is for the set of default flavor texts.

Development mode text:
Screenshot from 2023-12-11 21-27-45

Default mode text:
Screenshot from 2023-12-11 21-40-39

### CODE REVIEW

**Review by sampoder — APPROVED** [review]
which fraudsters have you been talking to?

**Comment by polypixeldev** [comment]
sam bankman-fried is the only one that I'm able to say at the moment, but rest assured that I have taken every step to make sure that the results of the study are accurate!

File: app/services/flavor_text_service.rb [hunk 1]
```diff
@@ -485,6 +486,7 @@ def flavor_texts
       "<marquee scrollamount='5'>💸💸💸</marquee>".html_safe,
       "Wow, that’s a lot of money. Need some help carrying it?",
       "I would rather check my Facebook than face my checkbook.",
+      "recommended by 100% of fraudsters",
```
Comment (maxwofford) [inline app/services/flavor_text_service.rb]: Want to make this a dev flavor text instead?

Comment (polypixeldev) [reply app/services/flavor_text_service.rb]: sure, might not be right for a default flavor text lol - will make the change later today when I get to a computer

**Review by garyhtou — APPROVED** [review]
perfect! 💸

File: app/services/flavor_text_service.rb (hunk 1 above)
Comment (garyhtou) [reply app/services/flavor_text_service.rb]: thanks!

**Pull request merged**